telco-core/configuration/Makefile

```diff
@@ -0,0 +1,36 @@
+CLUSTER_COMPARE_DOWNLOAD_URL := $(shell curl -s https://api.github.com/repos/openshift/kube-compare/releases/latest | jq -r '.assets[] | select(.name? | match("kube-compare_linux_amd64")) | .browser_download_url')
+HELM_CONVERT_DOWNLOAD_URL := $(shell curl -s https://api.github.com/repos/openshift/kube-compare/releases/latest | jq -r '.assets[] | select(.name? | match("kube-compare_addon_tools_linux_amd64")) | .browser_download_url')
+CLUSTER_COMPARE_DIR=/tmp
+CLUSTER_COMPARE_BIN=$(CLUSTER_COMPARE_DIR)/kubectl-cluster_compare
+CLUSTER_COMPARE :=  $(if $(shell command -v kubectl-cluster_compare 2>/dev/null), $(shell command -v kubectl-cluster_compare), $(CLUSTER_COMPARE_BIN))
+
+# main target to check the differences between reference and RDS CRs. 
+.PHONY: check
+check: compare_crs missing_crs
+
+
+.PHONY: compare_crs
+compare_crs: $(CLUSTER_COMPARE_BIN)
```

Comment:
The dependency should be `$(CLUSTER_COMPARE_BIN)` because `$(CLUSTER_COMPARE)` is a variable. If kube-compare tool is pre-installed, `CLUSTER_COMPARE` returns path name, and it could not be referred as `@$` in later target.